doc/cookbook/source/examples/regression/krr_nystrom.rst

```diff
@@ -0,0 +1,69 @@
+===============================
+Nyström Kernel Ridge Regression
+===============================
+
+The Nyström method is a technique for increasing the performance of kernel methods
+by calculating a lower rank approximation to the kernel matrix. It originally
+appeared in 1930 in the approximation of integral equations.
+
+The approximation is achieved by projecting the data matrix on a subset of data
+points, resulting in a linear system that is cheaper to solve. When applied to
+ridge regression, the following approximate system is solved
+
+.. math::
+    {\bf \alpha} = (\tau K_{m,m} + K_{m,n}K_{n,m})^+K_{m,n} {\bf y}
+
+where :math:`K_{n,m}` is a submatrix of the kernel matrix :math:`K`
+containing all n rows and the m columns
+corresponding to the m chosen training examples, :math:`K_{m,n}` is its
+transpose and :math:`K_{m,m}` is the submatrix with the m rows and columns
+corresponding to the training examples chosen. :math:`+` indicates the
+Moore-Penrose pseudoinverse. The complexity is :math:`O(m^2n)`.
```

Comment:
I think it is O(m^3 + m^2n), and you should also say that this is for computation.
I would also give memory requirements, and then compare both with the original KRR costs in memory and computation